tests/gui/search.goml

```diff
@@ -1,5 +1,6 @@
 // This tests basic search behavior.
 
+fail-on-js-error: true
```

Comment:
I'll make this value the default in `browser-ui-test` because it would have helped me catch this error much sooner.

Reply:
Done in https://github.com/rust-lang/mdBook/pull/2736.